source/dotnet/IntegrationTests/Hosts/ProcessManagerIntegrationTestHost.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright 2020 Energinet DataHub A/S
+//
+// Licensed under the Apache License, Version 2.0 (the "License2");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Azure.Messaging.ServiceBus;
+using Energinet.DataHub.Core.App.FunctionApp.Middleware.CorrelationId;
+using Energinet.DataHub.Wholesale.IntegrationTests.Mock;
+using Energinet.DataHub.Wholesale.ProcessManager;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using Microsoft.Extensions.Hosting;
+
+namespace Energinet.DataHub.Wholesale.IntegrationTests.Hosts;
+
+public sealed class ProcessManagerIntegrationTestHost : IDisposable
+{
+    private readonly IHost _processManagerHost;
+
+    private ProcessManagerIntegrationTestHost(IHost processManagerHost)
+    {
+        _processManagerHost = processManagerHost;
+    }
+
+    public static Task<ProcessManagerIntegrationTestHost> InitializeAsync(
```

Comment:
It doesn't initialize an existing "thing" but creates a new instance.
```suggestion
    public static Task<ProcessManagerIntegrationTestHost> CreateAsync(
```